pgml-dashboard/src/api/cms.rs

```diff
@@ -857,6 +862,50 @@ pub async fn careers_apply(title: PathBuf, cluster: &Cluster) -> Result<Response
     Ok(ResponseOk(layout.render(page)))
 }
 
+/// Redirect api to open-source
+#[get("/docs/api/<path..>")]
+pub async fn api_redirect(path: PathBuf) -> Redirect {
+    match path.to_str().unwrap() {
+        "apis" => Redirect::permanent("/docs/open-source/overview"),
+        "client-sdk/search" => {
```

Comment:
This should link to Korvus: https://postgresml.org/docs/open-source/korvus/guides/document-search